wagtail/images/fields.py

```diff
@@ -12,7 +12,9 @@
 
 def get_allowed_image_extensions():
     return getattr(
-        settings, "WAGTAILIMAGES_EXTENSIONS", ["gif", "jpg", "jpeg", "png", "webp"]
+        settings,
+        "WAGTAILIMAGES_EXTENSIONS",
+        ["gif", "jpg", "jpeg", "png", "webp", "avif"],
```

Comment:
```suggestion
        ["avif", "gif", "jpg", "jpeg", "png", "webp"],
```

otherwise you get failures in tests such as `test_select_format_flag_after_upload_form_error_bad_extension`

Reply:
God! this I would have never known ;-)